src/python/fuzzing/corpus_manager.py

```diff
@@ -40,7 +40,7 @@
 CORPUS_FILES_SYNC_TIMEOUT = 60 * 60
 LATEST_BACKUP_TIMESTAMP = 'latest'
 PUBLIC_BACKUP_TIMESTAMP = 'public'
-STATIC_CORPUS_GCS_PATH_SUFFIX = '_static'
+REGRESSIONS_GCS_PATH_SUFFIX = '_regressions'
```

Comment:
do we need to automatically create these buckets somehow?

Reply:
Nope, sub-dir get created automatically.

Reply:
ah, right! awesome